cirq-core/cirq/experiments/t1_decay_experiment.py

```diff
@@ -77,7 +77,12 @@ def t1_decay(
 
     var = sympy.Symbol('delay_ns')
 
-    sweep = study.Linspace(var, start=min_delay_nanos, stop=max_delay_nanos, length=num_points)
+    if min_delay_nanos == 0:
```

Comment:
use the original parameter to make it clear that this happens when no `min_delay` is supplied 

```py3
if min_delay is None
```